server/webapp/WEB-INF/rails.new/webpack/views/agents/environments_list_widget.js.msx

```diff
@@ -47,10 +47,19 @@ const EnvironmentsListWidget = {
   },
 
   view (vnode) {
-
     const ctrl = vnode.state;
     const args = vnode.attrs;
 
+    const spinner = (
+      <div class="agent-button-group-dropdown env-dropdown empty-container" onclick={ctrl.closeDropdown}>
```

Comment:
These are the same classes as the ones defined in `environmentDropDownClass`. Dedup it? `class` attribute accepts arrays (even with null and undefined objects), and they usually work better with optional class names.